AppKit/CPTableView.j

```diff
@@ -6400,18 +6400,12 @@ var CPTableViewDataSourceKey                = @"CPTableViewDataSourceKey",
 
 - (BOOL)setThemeState:(ThemeState)aState
 {
-    if (aState.isa && [aState isKindOfClass:CPArray])
```

Comment:
Seems fair for me. Can you just add a comment deprecated or something so we don't forget to remove it in the future